.github/workflows/build.yml

```diff
@@ -0,0 +1,17 @@
+name: Build for mobile
+on: [pull_request]
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./front
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: 'front/.tool-versions'
+      - run: npm ci
+      - run: npm run build
```

Comment:
You can add two more steps to the build workflow:
- Build for android
- Build for iOS

You can check the README.md at `front/README.md` to see the commands related commands.

Reply:
It's modified !